SPECS/terraform/terraform.spec

```diff
@@ -29,7 +29,7 @@ Source0:        https://github.com/hashicorp/terraform/archive/refs/tags/v%{vers
 Source1:        %{name}-%{version}-vendor.tar.gz
 %global debug_package %{nil}
 %define our_gopath %{_topdir}/.gopath
-BuildRequires:  golang >= 1.17.2
+BuildRequires:  golang >= 1.17.1
```

Comment:
according to terraform sources go version is 1.17.2 (see .go-version file in src). That said I bet it will build against any 1.17 version of go

Reply:
That's true, but I couldn't find any documentation saying that the golang version needs to be at least that version. I believe that file just indicates the version upstream uses to build.

Reply:
https://github.com/hashicorp/terraform/blob/main/.go-version